js/client-chat.js

```diff
@@ -567,11 +567,20 @@
 					for (var i=0, len=targets.length; i<len; i++) {
 						targets[i] = targets[i].trim();
 					}
-					switch (targets[0]) {
+					switch (targets.shift()) {
 					case 'add':
-						for (var i=1, len=targets.length; i<len; i++) {
-							highlights.push(targets[i].trim());
-						}
```

Comment:
So you want it to loop through ``targets`` twice? What would be the difference between that and ``Array.prototype.push.apply(highlights, targets)`` or ``highlights = highlights.concat(targets)``?